### PR TITLE
Update gifimage to TypeScript

### PR DESCRIPTION
**Pull request by berkbaski**

### CODE REVIEW

File: ui/gifimage/gifimage.android.ts [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+  get frameCount(): number {
+    return this.nativeObject.getNumberOfFrames();
+  }
+
+  get posterImage(): Image {
+    const bitmap = this.nativeObject.seekToFrameAndGet(0);
+    return new Image({ bitmap });
+  }
+
+  get instrinsicSize(): { width: number; height: number } {
```
Comment (cenkce) [inline ui/gifimage/gifimage.android.ts:61]: `{ width: number; height: number }` yerine Size kullanabilirsin

File: ui/gifimage/index.ts [hunk 2]
```diff
@@ -1,149 +1,180 @@
...
    * @android
    * @ios
-   * @readonly
-   * @property {Object} instrinsicSize
-   * @property {Number} instrinsicSize.width
-   * @property {Number} instrinsicSize.height
    * @since 3.2.0
    */
-  readonly instrinsicSize: { width: number; height: number };
+  instrinsicSize: { width: number; height: number };
```
Comment (cenkce) [inline ui/gifimage/index.ts:81]: { width: number; height: number } => Size

File: ui/gifimage/index.ts [hunk 3]
```diff
@@ -1,149 +1,180 @@
...
+  set loopCount(value: number) {
+    throw new Error('Method not implemented.');
+  }
+  get frameCount(): number {
+    throw new Error('Method not implemented.');
+  }
+  get posterImage(): Image {
+    throw new Error('Method not implemented.');
+  }
+  get instrinsicSize(): { width: number; height: number } {
```
Comment (cenkce) [inline ui/gifimage/index.ts:141]: Size

**Review by cenkce — CHANGES_REQUESTED** [review]

**Comment by berkbaski** [comment]
Could you recheck again? @cenkce

**Pull request merged**